copilot-studio/out-of-office-creator/README.md

```diff
@@ -0,0 +1,130 @@
+# Out of Office Creator
+
+**Out of Office Creator** is an AI-powered agent for **Microsoft Teams** and/or **Microsoft 365 Copilot** that helps users easily create and set Out of Office automatic reply messages for Outlook.
+
+![out-of-office-creator](./assets/out-of-office-creator.gif)
+
+The agent guides the user through a simple flow:
+
+- It collects key information such as the reason for absence, travel destination, start and end dates, and preferred tone of the message.
+- It uses AI to generate a complete, fluent Out of Office message based on the provided details.
+- After generating the message, the agent asks the user if they would like to set it as their automatic reply in Outlook.
+- Upon user confirmation, the agent automatically configure the Out of Office message.
+- Finally, the agent notifies the user that the message has been successfully set.
+
+![oof-creator-preview](./assets/oof-creator-preview.png)
+
+## Authors
+
+Author|Socials
+--------|---------
+Katerina Chernevskaya | [GitHub](https://github.com/Katerina-Chernevskaya/) - [LinkedIn](https://www.linkedin.com/in/katerinachernevskaya/)
+Artem Chernevskiy | [GitHub](https://github.com/artemchern) - [LinkedIn](https://www.linkedin.com/in/artem-chernevskiy/)
+
+## Prerequisites
+
+1. Agent in Copilot Studio (follow [this article](https://learn.microsoft.com/en-us/microsoft-copilot-studio/fundamentals-get-started?tabs=web#create-an-agent) to create your agent)
+2. Enable `Orchestration` in your agent
+![orchestration](./assets/orchestration.png)
+
+
+
+## Minimal path to awesome
+
+### Create the first - Prompt action
+
+1. Open your agent in **Copilot Studio**.
+2. Select **Actions**.
+3. Select **+ Add an action**.
+4. Select **+ New action** and select **New prompt**.
+5. Add the below prompt in the **Instruction** field:
```

Comment:
There are a couple of steps missing here. The new prompt dialog is a bit different: 

![image](https://github.com/user-attachments/assets/f46ca3c7-efa5-44f6-8e3c-e76c8533cf4d)

Reply:
@Katerina-Chernevskaya - if you could fix this - that would be great. The rest is all good to go!